tests/e2e/linalg_transform/benchmark_reduction.mlir

```diff
@@ -0,0 +1,21 @@
+// func.func @reduce(%arg : tensor<1x128x384xf32>) -> (tensor<1x128xf32>) {
+//   %cst = arith.constant -0.000000e+00 : f32
```

Comment:
This doesn't look like a `RUN` line.

Reply:
thanks, spurious change with rebasing and all ..